lib/moonchild.js

```diff
@@ -186,6 +195,16 @@ function getEditor() {
   return globalEditor;
 }
 
+function getChannel() {
```

Comment:
I'm not sure I understand why the channel is created here. It seems like it might make more sense to be in editor.js.